code/src/components/subintro/Subintro.tsx

```diff
@@ -0,0 +1,23 @@
+import "@/components/subintro/subintro.scss";
+import subintroData from "@/components/subintro/Subintro.constants";
+import SubintroElement from "@/components/subintroElement/SubintroElement";
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import Box from "@mui/material/Box";
+
+const Subintro = () => {
+  return (
+    <PageWrapper>
+      <Box className="spa-subintro">
+        {subintroData.map((element) => (
+          <SubintroElement
+            icon={element.icon}
+            key={element.id}
+            content={element.content}
+          />
+        ))}
```

Comment:
separate this map into variable and move it before return

Reply:
> separate this map into variable and move it before return

Its not necessary, because this components renders only Subintro data.